tests/performance-tests/include/performance-tests/services/s3/S3TestConfig.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#pragma once
+
+#include <aws/core/utils/memory/stl/AWSSet.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/memory/stl/AWSVector.h>
+
+namespace PerformanceTest {
+namespace Services {
+namespace S3 {
+namespace TestConfig {
+const Aws::Set<Aws::String> TestOperations = {"PutObject", "GetObject"};
```

Comment:
`Aws::Set`/`Aws::Vector`/`Aws::String` should not be used in this case because the static, and known at compilation time, they will not be added to at runtime and therefor do not need to be allocator aware.

something like
```cpp
struct Widget {
  const char* name;
  const int value;
};

namespace {
  const std::array<const char*, 4> my_array{"foo", "bar"};
  const std::array<const Widget, 2> m_widgets{{"name", 64}};
}
```
is more preferable as they are statically sized and defined